src/components/Link/Link.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import classnames from 'classnames'
+
+type StyledLinkProps = {
+  variant?: 'external' | 'unstyled'
+  className?: string
+}
+
+type LinkProps = {
+  href: string
+  children: React.ReactNode
```

Comment:
Added these back.  Trying to follow the pattern of [requiring props that are fundamental to the element](https://github.com/trussworks/react-uswds/blob/develop/docs/adding_new_components.md#props).  

Without these props `<Link className="usa-button" variant="unstyled" />` is valid  code. It compiles without warning ... and a regular anchor tag in our app would warn about this for missing `href`  and `children` (since we use jsx-a11y to check valid anchor tags).